playbooks/roles/ocp-upgrade/README.md

```diff
@@ -11,11 +11,12 @@ Requirements
 Role Variables
 --------------
 
-| Variable      | Required | Default | Comments                                                      |
-|---------------|----------|---------|---------------------------------------------------------------|
-| upgrade_image | no       | ""      | OCP upgrade image                                             |
-| pause_time    | no       | 90      | Pauses playbook execution for a set amount of time in minutes |
-| delay_time    | no       | 600     | Number of seconds to wait before starting to poll             |
+| Variable        | Required | Default    | Comments                                                      |
+|-----------------|----------|------------|---------------------------------------------------------------|
+| upgrade_version | no       | ""         | Set to a specific version eg. 4.5.4                        |
+| upgrade_channel | no       | stable-4.5 | Set to channel having required upgrade version available for cluster upgrade (stable-4.x, fast-4.x, candidate-4.x) |
```

Comment:
Let us not have default channel here. It is a pain to update this for future releases. Keep it "" for now with examples.